.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,42 @@
-### Description
+## Description
+<!-- Please include a summary of the change, including which issue it fixes or what feature it adds. Include relevant motivation, context and documentation as appropriate. List dependencies that are required for this change, if any. -->
 
-### Test Plan
-<!-- Please provide us with clear details for verifying that your changes work. -->
+## Type of Change
+- [ ] New feature
+- [ ] Bug fix
+- [ ] Performance improvement
+- [ ] Refactoring
+- [ ] Dependency update
+- [ ] Documentation update
+
+## Which Component or System Does This Change Impact?
+- [ ] Validator Node
+- [ ] Full Node (API, Indexer, etc.)
+- [ ] Move/Aptos Virtual Machine
+- [ ] Aptos Framework
+- [ ] Aptos CLI/SDK
+- [ ] Developer Infrastructure
+- [ ] Other (specify)
+
+## How Has This Been Tested?
+<!-- Please ensure that the functionality introduced by this change is well tested and verified to work as expected. Ensure tests cover both happy and unhappy paths. -->
+
+- [ ] Test A
+- [ ] Test B
```

Comment:
can we remove the Test A + B example? Feels kinda something that I would constantly need to delete.